ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/CCodeGenerator.java

```diff
@@ -104,8 +104,11 @@ public void generateSource(
     }
 
     public void generateStaticFiles() throws IOException {
-        final Path path = Paths.get("src/ds3_request.h");
-        processTemplate(null, "other-templates/ds3_request_h.ftl", fileUtils.getOutputFile(path));
```

Comment:
Since these templates are static, as in there's no auto generation going on, there's no object to inject into the template